orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/strategies/CustomStrategy.groovy

```diff
@@ -46,6 +46,13 @@ class CustomStrategy implements Strategy {
       parentStageId                          : stage.id
     ]
 
+    if (stage.context.image) {
```

Comment:
Is the imageID available here? I remember the reason for the code in the start pipeline task to handle the image Name is because the clone asg injects an image name. feels redundant.